src/test/librbd/test_Groups.cc

```diff
@@ -480,3 +480,255 @@ TEST_F(TestGroup, add_snapshotPP)
   ASSERT_EQ(0, rbd.group_snap_remove(ioctx, group_name, snap_name));
   ASSERT_EQ(0, rbd.group_remove(ioctx, group_name));
 }
+
+TEST_F(TestGroup, snap_get_info)
+{
+  REQUIRE_FORMAT_V2();
+
+  std::string pool_name2 = get_temp_pool_name("test-librbd-");
+  ASSERT_EQ(0, rados_pool_create(_cluster, pool_name2.c_str()));
+
+  rados_ioctx_t ioctx;
+  rados_ioctx_create(_cluster, _pool_name.c_str(), &ioctx);
+
+  rados_ioctx_t ioctx2;
+  rados_ioctx_create(_cluster, pool_name2.c_str(), &ioctx2);
+
+  const char *gp_name = "snap_group";
```

Comment:
Make these names unique at least across the file, see commit 4c69370f6b0e4c04e77fde67a9c199af6ea13649.